src/dropdown/dropdown.spec.ts

```diff
@@ -274,6 +274,28 @@ describe('ngb-dropdown-toggle', () => {
          expect(dropdownEl).toHaveCssClass('open');
        });
      })));
+  it('should have css class dropdown on div and dropdown-toggle on button',
```

Comment:
Right, but those are hosts of 2 different directives and we don't need to test them together!

IMO the whole change in this PR should boil down to sth like this: https://github.com/ng-bootstrap/core/commit/7dccb0c37d14c16a793f50d6e8de739952ab05fb

Could you update this PR accordingly?